### ISSUE TITLE
Reigniting Community: Coffee with The Carpentries Executive Director (19 February 2024) - *Discussing continuing work*

### ISSUE THREAD

**Issue by kariljordan** [carpentries/conversations#59]
_**Reigniting Community: Coffee with The Carpentries Executive Director,**_ @kariljordan is a series of six monthly open discussions to be hosted by Kari from January until June 2024 under the theme of 'Reigniting Community'.

![Reigniting community session announcement_February 2024](https://github.com/carpentries/conversations/assets/132367843/6a565ddd-0235-42cc-bc33-38d7f37146fb)

_**20 February Session Focus**_: The focus of this community session and accompanying GitHub conversation issue, the second one in the series, is to engage with the community on [continuing work](https://carpentries.org/blog/2024/02/changes-to-support-from-the-core-team-at-the-carpentries/) following the recent layoffs and [the CT members supporting that work](https://carpentries.org/blog/2024/01/reintroducing-the-carpentries-core-team/). 

_**This GitHub conversation issue**_: Kari will provide a summary of the discussions from this session in this conversation issue, including her answers to some of the questions that were raised at the session. If you missed the session and would like to pose a question or you attended it but have additional questions, please add them as comments below.

This conversation issue will remain open and Kari will actively engage with your questions and comments until we open the conversation issue for next month's _Reigniting Community_ session, which will focus on a different topic to be announced closer to the time.

**Comment by kariljordan** [comment]
**Summary of Coffee with The Carpentries Executive Director on 19 February 2024**

The second coffee with @kariljordan included seven attendees from the community. We began with a welcome, introductions, and an overview of the Code of Conduct.

@kariljordan shared a short [presentation](https://docs.google.com/presentation/d/1OMD82CXDgkLeN7QuZEZUjuLj3Fl2CL7r_ih_XXLiQlA/edit#slide=id.g2acff3a7536_0_80) offering a re-introduction to our nine member staff and the programming we'll support. We discussed what programming will be supported by the community moving forward, and programming that will sunset. Everyone is encouraged to review the presentation and consider the following questions:

1. What community-supported programming are you interested in learning more about?
2. How can we creatively celebrate the programming we are sunsetting, and what strategies should we implement to determine when and how to reintroduce it in the future?

A special thanks to Liz Stokes for providing a [mental model](https://drive.google.com/file/d/1M-HfjcEuHV-gJkNdvNSC3Of57PrMpLIa/view?usp=drive_link) of this discussion. Thank you to everyone who attended the session. The community is welcome to add more questions and comments to this issue.
